engine/src/main/scala/cromwell/engine/backend/jes/GenomicsFactory.scala

```diff
@@ -11,9 +11,8 @@ import cromwell.filesystems.gcs.GoogleConfiguration
 @deprecated(message = "This class will not be part of the PBE universe", since = "May 2nd 2016")
```

Comment:
Not sure what you mean, I didn't write this.  Certainly this class shouldn't stay in the engine, but moving it will probably need to wait until the old JES backend gets ripped out.